css/mobile.css

```diff
@@ -0,0 +1,227 @@
+        grid-column-gap: 53px;
+        grid-row-gap: 25px;
+;
+    }
+
+    .portfolio{
+        padding: 41px 32px 67px;
+        background-color: #2D303A;
+        border-bottom: 5px solid #323746;
+
+
+    }
+
+    .portfolio-images{
+        margin-top: 27px;
+        margin-left: 1px;
+        display: grid;
+        grid-template-columns: repeat(2,40vw);
+        grid-column-gap: 5vw;
+        grid-row-gap: 1vh;
+    }
+    .portfolio-images__component{
+        height: 12vh;
+
+        background-image: url("../images/portfolio-1.png");
+
+    }
+    .portfolio-sort{
+        margin-top: 14px;
+        margin-left: 0;
+    }
+    .portfolio-description-title{
+        margin-left: 0;
+    }
+    .footer{
+        padding: 24px 0 19px;
+        display: flex;
+        background: #2D303A;
+        justify-content: space-between;
+
+    }
+    .footer-copyright{
+
+        margin-left: 41px;
+        font-family: Lato;
+        font-style: normal;
+        font-weight: 900;
+        font-size: 13px;
+        line-height: 18px;
+        display: flex;
+        align-items: center;
+        color: #666D89;
+
+    }
+    .footer-navigation{
+
+        margin-left: 0;
+    }
+    .sheet{
+        display: none;
+        top: 0px;
+        position: absolute;
+        width: 100%;
+        height: 278vh;
+        background: black;
+        opacity: 0.2;
+    }
+    .sidebar{
+        top: 0px;
+        position: absolute;
+        width: 68vw;
+        color: #F06C64;
+        height: 278vh;
+        background: #2D303A;
+        flex-direction: column;
+        padding: 26px 20px;
+        display: none;
+    }
+    .sidebar-button{
+        display: block;
+        width: 25px;
+        height: 20px;
+        background: none;
+        border: none;
+        transform: rotate(90deg);
+    }
+    .sidebar-logo{
+        width: 92px;
+        height: 24px;
+        margin-left: 26px;
+        margin-top: 3px;
+        background-size: contain;
+        background-repeat: no-repeat;
+        background-image: url("../images/logo.png");
+    }
+    .sidebar-header{
+        display: flex;
+    }
+    .sidebar-nav{
+        margin-top: 180px;
+
+    }
+    .sidebar-nav-list{
+        padding: 0;
+        flex-direction: column;
+        list-style: none;
+        display: flex;
+    }
+    .sidebar-nav-list__component:not(:first-child){
+        margin-top: 28px;
+
+    }
+    .sidebar-nav-list__component-link{
+        font-family: Lato;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 24px;
+        line-height: 29px;
+        letter-spacing: -0.035em;
+        color: #FFFFFF;
+        margin-left: 51px;
+    }
+    .sidebar-nav-list__component:first-child {
+        list-style: none;
+        color: #F06C64;
+    }
+    .sidebar-nav-list__component:first-child .header-nav-list__component-link{
+        color: #F06C64;
+    }
+    .sidebar-nav-list__component:nth-child(2){
+        margin-left: 0;
```

Comment:
just add another class for these elements, and assign common styles there, in this case margin-left: 0;

Reply:
Added class instead of styles in .js